docs/reference/release-notes/highlights.asciidoc

```diff
@@ -13,11 +13,30 @@ For detailed information about this release, see the <<es-release-notes>> and
 
 // Use the notable-highlights tag to mark entries that 
 // should be featured in the Stack Installation and Upgrade Guide:
+
 // tag::notable-highlights[] 
-// [discrete]
-// === Heading
-//
-// Description. 
+[discrete]
+=== Better space efficiency for `keyword`, `match_only_text`, and `text` fields
+
+The inverted index switched to a new encoding that is more space-efficient.
+This change will benefit `keyword` fields, `match_only_text` fields and, to a
+lesser extent, `text` fields. On the dataset of application logs that the {es}
+team uses for nightly benchmarks, this translated into a 14.4% reduction of the
```

Comment:
I don't _think_ we need to mention nightly benchmarks, but ignore this if I'm wrong.
```suggestion
lesser extent, `text` fields. In our benchmarks using application logs, this translated into a 14.4% reduction of the
```